src/tracer/MOM_tracer_registry.F90

```diff
@@ -585,12 +621,14 @@ subroutine register_tracer_diagnostics(Reg, h, Time, diag, G, GV, US, use_ALE, u
 
     ! KPP nonlocal term diagnostics
     if (use_KPP) then
-      Tr%id_net_surfflux = register_diag_field('ocean_model', Tr%net_surfflux_name, diag%axesT1, Time, &
-          Tr%net_surfflux_longname, trim(units)//' m s-1', conversion=Tr%conc_scale*GV%H_to_m*US%s_to_T)
+      Tr%id_net_surfflux = register_diag_field('ocean_model', Tr%net_surfflux_name, diag%axesT1, &
+          Time, Tr%net_surfflux_longname, trim(units)//' m s-1', &
```

Comment:
Whenever possible, I strongly prefer that the `units` and `conversion` arguments in calls to `register_diag_field()` are kept on the same line, as this makes it possible to verify their consistency using simple `grep` commands rather than having to use a more complicated code parser.  Please consider undoing the revisions to break up lines 624-626, 629-631, 590-591, 440-441, 446-447, 386-386, 391-392, even if this leads to lines that are longer than 100 characters long (so long as they are less than the MOM6 hard limit of 120 characters).

Reply:
I suggest adding the following to [https://github.com/NOAA-GFDL/MOM6/wiki/Code-style-guide#line-length](url)

o An exception is that the units and conversion arguments in calls to register_diag_field() should be kept on the same line, even if this leads to lines that are longer than 100 characters long (providing they are less than the MOM6 hard limit of 120 characters).